spec/lib/pull_request_spec.rb

```diff
@@ -4,6 +4,25 @@
 RSpec.describe PullRequest do
   before { set_up_mock_token }
 
+  def fake_commit
+    <<~TEXT
+      Bump govuk_publishing_components from 35.7.0 to 35.8.0
```

Comment:
We should probably find a way of testing the alternate commit message format too. I think an explicit test like `it "only looks at the dependencies listed in the commit message"` would be a good idea.